cmd/hubd/main.go

```diff
@@ -97,13 +97,13 @@ var (
 				Key:      "dns.token",
 				DefValue: "",
 			},
-			"emailFrom": {
-				Key:      "email.from",
-				DefValue: "Hub <verify@email.textile.io>",
+			"emailInviteTmpl": {
+				Key:      "email.invite_template",
+				DefValue: "2",
 			},
-			"emailDomain": {
-				Key:      "email.domain",
-				DefValue: "email.textile.io",
+			"emailConfirmTmpl": {
+				Key:      "email.confirm_template",
+				DefValue: "3",
```

Comment:
email template values, they can be updated on the service (2 and 3 are currently usable ones), but felt like config flags not hard-coded values in the code